### PR TITLE
Update default docker image name and README.md

### PR DESCRIPTION
**Pull request by chenquanzhao**

### CODE REVIEW

File: docker_image/README.md [hunk 1]
```diff
@@ -1,45 +1,45 @@
...
 ```
 
-Then we can connect to the PostgreSQL successfully inside the container (retry the connecting command again).
+Then we can connect to the VidarDB successfully (retry the connecting command again). 
 
-## Building your own Docker image
+## Build your own Docker image
 
-You may want to create your custom Docker image. Do it by '1-click':
+You may want to create your custom Docker image. Do it by '1-click' in the `vidardb` directory:
```
Comment (xingweitian) [inline docker_image/README.md]: Maybe "in the root directory of `vidardb` repository" is more clear.

File: docker_image/Dockerfile [hunk 2]
```diff
@@ -34,4 +34,4 @@ RUN set -xe && apt-get update && \
 COPY docker-entrypoint.sh install-madlib.sh health-check.sh /usr/local/bin/
 
 # Add health check
-HEALTHCHECK CMD health-check.sh
\ No newline at end of file
+HEALTHCHECK --start-period=5s CMD health-check.sh
```
[the remark this answers is not in the document]
Comment (chenquanzhao) [reply docker_image/Dockerfile]: okay

File: docker_image/README.md [hunk 3]
```diff
@@ -1,45 +1,45 @@
...
-For the users who don't have the PostgreSQL client installed:
+For the others who have the PostgreSQL client installed:
 
 ```sh
-docker exec -it postgresql /bin/bash
+psql -h 127.0.0.1 -p 5432 -U postgres
 ```
 
-Then we can connect to the PostgreSQL successfully inside the container (retry the connecting command again).
+Then we can connect to the VidarDB successfully (retry the connecting command again).
```
[the remark this answers is not in the document]
Comment (chenquanzhao) [reply docker_image/README.md]: Got

File: docker_image/README.md (hunk 1 above)
Comment (chenquanzhao) [reply docker_image/README.md]: okay

**Comment by chenquanzhao** [comment]
@xingweitian Sorry to rebase the previous commit, and I have updated the README and please review again.

**Pull request merged**